lib/streamlit/runtime/fragment.py

```diff
@@ -163,7 +163,7 @@ def wrap(*args, **kwargs):
 
         ctx = get_script_run_ctx()
         if ctx is None:
-            return
+            return None
```

Comment:
I think the actual reported issue in https://github.com/streamlit/streamlit/security/code-scanning/9343 was that the return statement is not at the end of the function, for which I opened a PR here: https://github.com/streamlit/streamlit/pull/9281
But adding `None` here explicitly also makes sense 👍